src/network/tarcap/new_taraxa_capability.hpp

```diff
@@ -78,33 +79,29 @@ class TaraxaCapability : virtual dev::p2p::CapabilityFace {
   //  // METHODS USED IN REAL CODE
   //  void start();
   //  bool isStarted();
-  //  std::list<dev::p2p::NodeEntry> getAllNodes() const;
-  //  unsigned getPeerCount();
-  //  unsigned getNodeCount();
   //  Json::Value getStatus();
-  //  std::vector<dev::p2p::NodeID> getAllPeers() const;
-  //  void onNewBlockVerified(std::shared_ptr<DagBlock> const &blk);
-  //  void onNewTransactions(std::vector<taraxa::bytes> transactions);
-  //  void restartSyncingPbft(bool force = false);
-  //  void onNewPbftBlock(std::shared_ptr<PbftBlock> const &pbft_block);
-  //  bool pbft_syncing();
-  //  void onNewPbftVotes(std::vector<Vote> votes);
-  //  void broadcastPreviousRoundNextVotesBundle();
-  //  void sendTransactions(dev::p2p::NodeID const &_id, std::vector<taraxa::bytes> const &transactions);
-  //
-  //  // METHODS USED IN TESTS ONLY
-  //  void sendBlock(dev::p2p::NodeID const &id, DagBlock const &blk);
-  //  void sendBlocks(dev::p2p::NodeID const &id, std::vector<std::shared_ptr<DagBlock>> blocks);
-  //  void setPendingPeersToReady();
-  //  dev::p2p::NodeID getNodeId();
-  //  int getReceivedBlocksCount();
-  //  int getReceivedTransactionsCount();
-  //  std::shared_ptr<TaraxaPeer> getPeer(dev::p2p::NodeID const &id);
-  //
-  //  // PBFT
-  //  void sendPbftBlock(dev::p2p::NodeID const &id, PbftBlock const &pbft_block, uint64_t const &pbft_chain_size);
-  //  void sendPbftVote(dev::p2p::NodeID const &id, Vote const &vote);
-  //  // END METHODS USED IN TESTS ONLY
+  std::vector<dev::p2p::NodeID> getAllPeersIDs() const { return peers_state_->getAllPeersIDs(); }
+  size_t getPeerCount() { return peers_state_->getPeersCount(); }
+  void restartSyncingPbft(bool force = false) { return syncing_state_->restartSyncingPbft(force); }
+  bool pbft_syncing() const { return syncing_state_->is_pbft_syncing(); }
+  void onNewBlockVerified(std::shared_ptr<DagBlock> const &blk);
+  void onNewTransactions(std::vector<taraxa::bytes> transactions);
+  void onNewPbftBlock(std::shared_ptr<PbftBlock> const &pbft_block);
+  void broadcastPreviousRoundNextVotesBundle();
+  void sendTransactions(dev::p2p::NodeID const &id, std::vector<taraxa::bytes> const &transactions);
+
+  // METHODS USED IN TESTS ONLY
+  void sendBlock(dev::p2p::NodeID const &id, DagBlock const &blk);
+  void sendBlocks(dev::p2p::NodeID const &id, std::vector<std::shared_ptr<DagBlock>> blocks);
+  void setPendingPeersToReady() { return peers_state_->setPendingPeersToReady(); }
+  size_t getReceivedBlocksCount() const { return test_state_->test_blocks_.size(); }
+  size_t getReceivedTransactionsCount() const { return test_state_->test_transactions_.size(); }
+  std::shared_ptr<TaraxaPeer> getPeer(dev::p2p::NodeID const &id) { return peers_state_->getPeer(id); }
```

Comment:
maybe we could keep functions implementations in cpp files even if those are only 1 liners ?

Reply:
I have checked google style guide, nothing about that is there so let's have them in cpp